src/components/ExerciseTwoForm.js

```diff
@@ -0,0 +1,78 @@
+// TODO: Exercise 2 Modify me
+
+import React, { PropTypes } from 'react';
+import { Field } from 'redux-form';
+import InputField from './Form/InputField';
+import CheckboxField from './Form/CheckboxField';
+
+const propTypes = {
+    handleSubmit: PropTypes.func.isRequired,
+    submitSucceeded: PropTypes.bool.isRequired,
+    ticked: PropTypes.bool,
+    name: PropTypes.string,
+};
+
+const defaultProps = {
+    ticked: false,
+    name: '',
+};
+
+const ExerciseTwoForm = ({ handleSubmit, submitSucceeded, ticked, name }) => (
+    <div>
+        <div>
+            <h3>Exercise 2</h3>
+            <p>
+                I need to be a form that takes an input (using InputField component) name
+                and a checkbox (using a CheckboxField component that you have to create).<br />
+                Label for input is "Your name:" and label for checkbox is
+                "I agree with these Terms and Conditions?"<br />
+                Please make sure the name is required. It must be displayed in the input as capital
+                letters but it must be stored in lowercase in the state.<br />
+                Also raise a warning if it is not only alphabetic string.<br />
+                Note: There is no need to modify the reducers or the actions.
+            </p>
+            <p>
+                TODO: Implement me in /components/Form/CheckboxField.js &amp;<br />
+                Modify /containers/ExerciseTwoForm.js &amp; /components/Form/InputField.js
```

Comment:
I don't think anything needs to be changed in `/containers/ExerciseTwoForm.js` maybe remove this from the sentence - could be confusing if it stays there. I believe it's actually `/components/ExerciseTwoForm.js`  that will be modified :)